ql/src/java/org/apache/hadoop/hive/ql/TaskQueue.java

```diff
@@ -46,7 +46,7 @@
  */
 public class TaskQueue {
 
-  private static final Logger LOG = LoggerFactory.getLogger(Driver.class.getName());
+  private static final Logger LOG = LoggerFactory.getLogger(TaskQueue.class.getName());
```

Comment:
nit:

Does not require a `TaskQueue.class.getName()`, `TaskQueue.class` is good enough.